src/header/toolbar/toolbar.component.html

```diff
@@ -4,11 +4,17 @@
     </span>
     <h1>{{title | async}}</h1>
     <span class="toolbar-spacer"></span>
-    <span class="global-toolbar-buttons">
-        <ng-content></ng-content>
-    </span>
-    <span class="toolbar-buttons">
-        <ng-template [cfToolbarHost]="portal | async">
-        </ng-template>
-    </span>
+    <ng-container [ngSwitch]="globalButtonAlign | async">
```

Comment:
instead of changing the order in the document, we can use flexbox to either reverse the order or explicitly set the flexbox order.